openwisp_monitoring/check/classes/config_applied.py

```diff
@@ -0,0 +1,35 @@
+from swapper import load_model
+
+from ...device.utils import SHORT_RP
+from ..settings import CONFIG_CHECK_MAX_TIME
+from .base import BaseCheck
+
+AlertSettings = load_model('monitoring', 'AlertSettings')
+
+
+class ConfigApplied(BaseCheck):
+    def check(self, store=True):
+        # If the device is down do not run config applied checks
+        if self.related_object.monitoring.status == 'critical':
+            return
+        if not hasattr(self.related_object, 'config'):
+            return
+        result = 1 if self.related_object.config.status == 'applied' else 0
```

Comment:
this could be rewritten as:

```python
int(self.related_object.config.status == 'applied')
```

not sure which one is more readable, I think this form is compact and readable